### ISSUE TITLE
12.5 conversion error

### ISSUE THREAD

**Issue by Swifty819** [MisterMonstr/2hu_steam-exe_to_disc-exe#2]
Trying to convert 12.5 using these files on Win10 64 bit results in a "This App Can't Run on Your PC" error from the operating system.

**Comment by MisterMonstr** [comment]
Apologies for the delay; I don't know how I ended up with a diff file that was only a couple of bytes long the first time. This new one I just pushed should do the trick.

**Comment by Swifty819** [comment]
Yeah that'd cause something like this for sure. It'll be a moderately long
time before I can verify though. My motherboard decided to start failing.
Would you like me to write back when I get around to this?

**Comment by MisterMonstr** [comment]
Yeah, let me know if it works, whenever you want to get around to it. I don't mind waiting a while.
